File: rmt/src/test/scala/de/flower/rmt/ui/page/venues/manager/VenuesPageTest.scala
package de.flower.rmt.ui.page.venues.manager

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class VenuesPageTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startPage(new VenuesPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/page/venues/manager/VenuesPageTest.java
package de.flower.rmt.ui.page.venues.manager;

import de.flower.rmt.test.AbstractRMTWicketIntegrationTests;
import org.testng.annotations.Test;


public class VenuesPageTest extends AbstractRMTWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new VenuesPage());
        wicketTester.dumpPage();
    }
}